app/models/footnote.rb

```diff
@@ -53,7 +53,7 @@ def footnote_description
                                   right_key: [:meursing_table_plan_id, :meursing_heading_number]
 
 
-  delegate :description, :formatted_description, to: :footnote_description
+  delegate :description, :formatted_description, to: :footnote_description, allow_nil: true
```

Comment:
@alexey-klimuk still not sure, why would a footnote be found (in the timemachine) if its description is nil? 

Its not really valid without a description. Looking again at the migration, we should not be perhaps updating the data in the db, but adding a new record which 'ends' the old records. So adding new update records which adds the validity end date.